modules/data.remote/R/call_MODIS.R

```diff
@@ -26,66 +29,118 @@
 ##' }
 ##' 
 ##' @author Bailey Morrison
-##'  
-call_MODIS <- function(outfolder = ".", start_date, end_date, lat, lon, size = 0, product, band, band_qc = "", band_sd = "", package_method = "MODISTools") {
-  
+##'
+call_MODIS <- function(outfolder = "", start_date, end_date, lat, lon, size = 0, product, band, band_qc = "", band_sd = "", siteID = NULL, package_method = "MODISTools", QC_filter = FALSE, progress = TRUE) {
```

Comment:
Do you like blank or should we instead set these to NULL?  Suppose I dont really care either way

Reply:
I like it blank because if you don't want to save the data, using "" for outfielder makes it so it won't save the data anywhere.